service/static/install.sh

```diff
@@ -1,6 +1,24 @@
-#!/bin/sh
+#!/bin/bash
```

Comment:
We should be able to leave it ash `/sh`, don't seen any bashism.

Reply:
Done.